adaptive/runner.py

```diff
@@ -467,6 +467,8 @@ class AsyncRunner(BaseRunner):
         The overhead in percent of using Adaptive. This includes the
         overhead of the executor. Essentially, this is
         ``100 * (1 - total_elapsed_function_time / self.elapsed_time())``.
+    block_until_done : callable
```

Comment:
The name and the description is vague: it is unclear what's being blocked.

The original ioloop method name is one better alternative.

Reply:
I don't think that `run_until_complete` is a good name neither. The `Runner` will always run until it is complete, however, it won't block the kernel.

Reply:
Perhaps. `wait`/`wait_for_goal` are some alternatives.

Just for my personal understanding: why do we explicitly list methods and their types here? Are there methods that aren't callable?

Reply:
The only reason I added it there is because the other were already there.

I don't really get why it's there in the first place though.